pkg/morserino_com/listen_com.go

```diff
@@ -24,67 +24,126 @@ THE SOFTWARE.
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"strings"
+	"testing/iotest"
 
-	"morserino_display/pkg/morserino_console"
-
+	"github.com/on4kjm/morserino_display/pkg/morserino_console"
 	"go.bug.st/serial"
 )
 
+// Channel used to forward data received on the serial port to the display modules
+var MessageBuffer = make(chan string, 10)
+
```

Comment:
:thinking: 

I was pretty sure you didn't want to use a package level channel for this message buffer. Are you sure you want to land this?

Reply:
Agreed. This is the "stubb" of what I planned to do with channels. And that we discussed on Friday.